pypeit/core/gui/identify.py

```diff
@@ -190,6 +196,10 @@ def initialise(cls, arccen, slits, slit=0, par=None, wv_calib_all=None, wavelim=
             If a best-fitting solution exists, and you wish to load it, provide the wv_calib dictionary.
         wavelim : :obj:`list`, None, optional
             A two element list containing the desired minimum and maximum wavelength of the linelist
+        test : bool, optional
+            If True, this is a unit test
+        fwhm : float, optional
+            FWHM of arc lines in pixels
```

Comment:
`nonlinear_counts` and `pxtoler` seem missing in this doc

Reply:
added